### PR TITLE
Update additional images xml requirements

### PR DESCRIPTION
**Pull request by srekoble**

### CODE REVIEW

File: source/localizable/feedspec/index.en.html.md.erb [hunk 1]
```diff
@@ -266,7 +266,8 @@ photos of the same product from different angles or with a different
 presentation or combinations of the above add value to the presentation
 of the product in <%= settings.site_name.capitalize %>.
 
-This field is optional but recommended. It can occur multiple times
+This field is optional but recommended, except of fashion categories
```
Comment (Zorbash) [inline source/localizable/feedspec/index.en.html.md.erb]: `except for`

https://www.usingenglish.com/forum/threads/198474-except-of

File: source/localizable/feedspec/index.el.html.md.erb [hunk 2]
```diff
@@ -286,9 +286,12 @@ thumbnail.
...
 
-Το πεδίο είναι προαιρετικό αλλά συνίσταται η χρήση του. Μπορεί να εμφανίζεται
-πολλές φορές σε ένα feed, μιά φορά για κάθε διαφορετική φωτογραφία.
-Ισχύουν οι ίδιοι περιορισμοί με το [Image Link](#image-link)
+Το πεδίο είναι προαιρετικό αλλά συνίσταται η χρήση του, με εξαίρεση τα προϊόντα
+που ανήκουν στις κατηγορίες της Μόδας, όπου η συμπλήρωση των επιπρόσθετων
+φωτογραφιών είναι υποχρεωτική, εφόσον το κατάστημα τις προβάλλει στη σελίδα
+των προϊόντων του. Μπορεί να εμφανίζεται πολλές φορές σε ένα feed, μια φορά
+για κάθε διαφορετική φωτογραφία. Ισχύουν οι ίδιοι περιορισμοί με
+το [Image Link](#image-link)
```
Comment (cvkef) [inline source/localizable/feedspec/index.el.html.md.erb]: End phrase with a fullstop `.`

File: source/localizable/csvspec/index.en.html.md.erb [hunk 3]
```diff
@@ -206,7 +206,8 @@ photos of the same product from different angles or with a different
 presentation or combinations of the above add value to the presentation
 of the product in <%= settings.site_name.capitalize %>.
 
-This field is optional but recommended. It can occur multiple times
+This field is optional but recommended, except of fashion categories
```
Comment (cvkef) [inline source/localizable/csvspec/index.en.html.md.erb]: `except for` as @Zorbash mentioned below.

File: source/localizable/feedspec/index.en.html.md.erb (hunk 1 above)
Comment (srekoble) [reply source/localizable/feedspec/index.en.html.md.erb]: amr @Zorbash 👑

File: source/localizable/csvspec/index.en.html.md.erb (hunk 3 above)
Comment (srekoble) [reply source/localizable/csvspec/index.en.html.md.erb]: fixed

File: source/localizable/feedspec/index.el.html.md.erb (hunk 2 above)
Comment (srekoble) [reply source/localizable/feedspec/index.el.html.md.erb]: fixed

File: source/localizable/feedspec/index.en.html.md.erb (hunk 1 above)
Comment (srekoble) [reply source/localizable/feedspec/index.en.html.md.erb]: fixed

**Pull request merged**